src/SignalR/clients/ts/FunctionalTests/scripts/run-tests.ts

```diff
@@ -289,6 +299,8 @@ function runJest(httpsUrl: string, httpUrl: string) {
 
         debug(`Functional Test Server has started at ${httpsUrl} and ${httpUrl}`);
 
+        await ensureWdInstalled();
```

Comment:
Is the workaround done later now? If yes, why?

Reply:
Yes it's later, couple reasons:
* Before it wasn't in an async context so would have had to use `execSync` which I'd prefer to avoid
* Now it's closer to the relevant section of code that actually needs `wd` installed correctly.